test/integration/statistic_js_test.rb

```diff
@@ -0,0 +1,13 @@
+require 'integration_test_helper'
+
+class StatisticIntegrationJSTest < IntegrationTestWithJavascript
+  test "add statistics" do
+    visit statistics_path
+    click_link "New Statistic"
+    fill_in "statistic_name", :with => "facter-version"
+    fail(page.body)
```

Comment:
This test is failing because the select box [1] hasn't got any options in the test environment. I'm not sure why?

[1] - https://github.com/theforeman/foreman/pull/3438/files#diff-8733198cbbf2a319fdb74576db385f73R7